exec/java-exec/src/test/java/org/apache/drill/TestBugFixes.java

```diff
@@ -221,4 +221,14 @@ public void testDRILL4884() throws Exception {
             .baselineRecords(baseline)
             .go();
   }
+
+  @Test
+  public void testDRILL5051() throws Exception {
+    testBuilder()
+        .sqlQuery("select count(1) as cnt from (select l_orderkey from (select id from cp.`tpch/lineitem.parquet` limit 2) limit 1 offset 1)")
```

Comment:
This query seems to hit SQL validation error " Column 'l_orderkey' not found in any table". 

I'm going to change column "id" to "l_orderkey" in the inner subquery, in order to skip this validation error.